docs/modules/server.rst

```diff
@@ -34,8 +34,3 @@ The hivemind.server module is organized as follows:
 .. autoclass:: ExpertBackend
     :members: forward, backward, apply_gradients, get_info, get_pools
     :member-order: bysource
-
```

Comment:
Please restore TaskPool to fix RTFD or remove hyperlinks to TaskPool from docs

/* yep, this is the only issue that caused RTFD to faul */